qt/scientific_interfaces/Inelastic/Corrections/ContainerSubtraction.cpp

```diff
@@ -205,8 +212,6 @@ void ContainerSubtraction::newSample(const QString &dataName) {
   // Remove old sample and fit curves from plot
   m_uiForm.ppPreview->removeSpectrum("Subtracted");
   m_uiForm.ppPreview->removeSpectrum("Sample");
-  // Remove the subtracted workspace as it is no longer valid
-  removeOutput();
```

Comment:
This private function is not used anymore so it is safe to delete its definition. Is there a reason to keep the definition?